recipes/seqan3/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "SeqAn3" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/seqan/{{ name|lower }}/releases/download/{{ version }}/seqan-{{ version }}-with-submodules.tar.gz
+  sha256: ba8f0fd5d19289f95823235b69fdc5a12aba3578292463eecf750722e8662bd3
+  patches:
+    - bioconda.patch
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  run:
+    - bzip2
```

Comment:
Mh, this does not make much sense imho. People (seqan users) still need to put zlib and co into their host-sections, so also have it during build-time - and this in turn puts them into the run-time section of the users-package. So its save to remove this imho.

Reply:
I can just remove the requirements section with the next update, if that is soon enough :)

Reply:
Sure :)